### DISCUSSION TITLE
Research on FhirProto  -  using Protocol Buffers and how we could leverage this on FHIR Core

### DISCUSSION THREAD

**Discussion by f-odhiambo** [opensrp/fhircore#300] [Ideas]
Clone and review the Proto lib  [link](https://github.com/google/fhir) to document its capabilities

**Comment by pld** [comment]
I don't see any urgency on this, when we discussed before HAPI protocols give us what we need and we can follow the SDK's lead on moving over to protobuf when it supports key features needed for the SDK.

Unassigning for now
